.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,24 @@
+name: lint_python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: sudo apt-get update
+      - run: sudo apt-get install --fix-missing ffmpeg python3-magic python3-soundfile
+      - run: pip install bandit black codespell flake8 isort mypy pytest pyupgrade safety
+      - run: bandit --recursive --skip B101,B301,B303,B311,B403 .
+      - run: black --check . || true
+      - run: codespell --ignore-words-list="tha" --skip="*/text_tests,*/misspelling.json" || true  # Until #51 lands
+      - run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+      - run: flake8 . --count --exit-zero --max-complexity=15 --max-line-length=90 --show-source --statistics
+      - run: isort --check-only --profile black . || true
+      - run: pip install torchvision -r requirements.txt
+      - run: pip install -e .
+      - run: mypy --ignore-missing-imports . || true
+      - run: python -m unittest discover -s augly/tests/image_tests/ -p "*" || true
```

Comment:
You shouldn't need the `|| true` here. I see the pytorch image test was failing, which I just fixed here: https://github.com/facebookresearch/AugLy/pull/59. Once that PR is landed, you can rebase & all image tests should pass.